server/routes/selectLocation.ts

```diff
@@ -37,13 +37,15 @@ export default function routes(router: Router): Router {
     const allLocations: Array<Location> = await prisonApi.getUserLocations()
     const selectedLocation = allLocations.find(location => location.locationPrefix === locationPrefix)
 
+    const subLocationChar = selectedLocation.subLocations ? '-' : ''
+
     if (!selectedLocation) {
```

Comment:
looks like `selectedLocation` can be absent so i'd move the `subLocationChar` line to be below this if block